packages/orchestration/src/exos/chain-hub-admin.js

```diff
@@ -56,6 +54,10 @@ export const prepareChainHubAdmin = (zone, chainHub) => {
           chainHub.getChainInfo('agoric'),
         );
         chainHub.registerChain(chainName, chainInfo);
+        if (!connectionInfo) {
+          console.log('no connection info for', chainName, 'assuming CCTP');
```

Comment:
assuming CCTP is sort of spike-mode code.
likewise `console.log` rather than `trace`

maybe something like:
```suggestion
          trace(chainName, 'registered without connection info');
```

If we want the design to be that CCTP is, to date, the only path other than IBC, perhaps we should assert `chainInfo.cctpDestinationDomain`.

Reply:
I dropped `cctpDestinationDomain` for cosmos chains, so it can't be required.